src/NearOrg/DataAvailabilityPage.jsx

```diff
@@ -497,8 +497,12 @@ return (
                     Drastically reduce your costs
                   </Text>
                   <Text>
-                    Storing calldata on NEAR Protocol is approximately 8000x cheaper than storing the same amount of
-                    data on Ethereum.
+                    Storing calldata on NEAR Protocol is approximately 8000x
+                    cheaper than storing the same amount of data on Ethereum.
+                    Harness the power of NEAR's efficient network, optimized for
+                    transactions, data storage, and blockchain functionality.
+                    Say goodbye to high fees and embrace a new era of
+                    cost-effective blockchain solutions.
```

Comment:
Made some updates:

Storing rollup calldata on NEAR Protocol is approximately 8000x cheaper than storing the same amount of data on Ethereum. Harness the power of NEAR's robust blockchain, optimized for speed, scalability, and cost.  Say goodbye to high DA fees and embrace a new era of cost-effective modular blockchain solutions.